### DISCUSSION TITLE
Spanish characters á ó é í ú not supported

### DISCUSSION THREAD

**Discussion by Aristo3d** [friction2d/friction#371] [Ideas]
Hi! Is there any plans for giving support to spanish characters with graphical accent á é í ó ú?

**Comment by rodlie** [comment]
Accents works fine here, what version (of Friction) and OS are you using?

**Comment by Aristo3d** [comment]
I'm trying 1.0.0 beta 2 in windows 11. In preview and export, the accented letter is changed with an ? sign. (it's my first time with the app).

**Reply by pgilfernandez** [reply]
> I'm trying 1.0.0 beta 2 in windows 11. In preview and export, the accented letter is changed with an ? sign. (it's my first time with the app).

I'm Spanish and I'm not experiencing any issue... maybe it's the font you are using.
Could you explain your problem sending pictures and telling us your OS?

Cheers

**Reply by rodlie** [reply]
> 1.0.0 beta 2 in windows 11

![friction-accent-win11-23h2](https://github.com/user-attachments/assets/6389d5a2-29b9-470f-ae63-33dceb4c26ce)

Friction 1.0.0 beta 2 on Windows 11 23H2.

**Comment by Aristo3d** [comment]
Hi, sorry for  the delay in replying. I'm in windows 11 home Versión 10.0.26100 compilación 26100. 

Accented characters are displayed correctly in the program, but when you preview or export the svg, the accented letter is changed and the font is changed (both opening  it with a web browser or in inskcape). No matter the font you pick. Happened the same with the stable version and the beta.

![Captura de pantalla 2024-12-17 181447](https://github.com/user-attachments/assets/25f5f9d5-cc27-4c6d-a0ea-f644c7f05276)

![Captura de pantalla 2024-12-17 181910](https://github.com/user-attachments/assets/5a5539b0-9176-4dfc-b33f-a7a353a54319)

**Reply by rodlie** [reply]
Thanks for your reply, that explains the issue better.

Will try to look into the issue this week.

**Reply by Aristo3d** [reply]
Thanks to  you for your amazing work!!!

**Comment by kaixoo12** [comment]
Same issue here.... wayland maybe?

Edit: actually not the same, I cnt input the accents into the text field box, they do not even type anything. Weird

**Reply by rodlie** [reply]
This was a packaging issue on Linux (fixed a long time ago) when we moved from Qt 5.12 to 5.15.

I assume you are using a custom build, if so then it's a local/distro issue, not Friction.
